### ISSUE TITLE
Extended / x-pack/metricbeat-macosM1-orka && darwin && aarch64 / TestData – github.com/elastic/beats/v7/x-pack/metricbeat/module/*

### ISSUE THREAD

**Issue by v1v** [elastic/beats#33036]
## Flaky Test

* **Test Name:** `Extended / x-pack/metricbeat-macosM1-orka && darwin && aarch64 / TestData – github.com/elastic/beats/v7/x-pack/metricbeat/module/`
* **Link:** Link to file/line number in github.
* **Branch:** https://github.com/elastic/beats/pull/32173
* **Artifact Link:** https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats/detail/PR-32173/5/pipeline/3289
* **Notes:** There are `19` Test failures for `x-pack/metricbeat` running on Macos M1

<img width="1721" alt="image" src="https://user-images.githubusercontent.com/2871786/189325347-58d43ccc-9d9d-49c2-81cb-e8af862c90d9.png">

### Stack Trace

```
10:42:59          	Test:       	TestPromHistogramToES/new_bucket_between_two_other_buckets_on_the_go
10:42:59      histogram_test.go:402: #1: {SampleCount:0x14000438bf0 SampleSum:0x14000438bf8 Bucket:[cumulative_count:1 upper_bound:0.09  cumulative_count:2 upper_bound:0.49  cumulative_count:13 upper_bound:0.99 ] XXX_NoUnkeyedLiteral:{} XXX_unrecognized:[] XXX_sizecache:0}
10:42:59      histogram_test.go:404: 
10:42:59          	Error Trace:	histogram_test.go:404
10:42:59          	Error:      	Not equal: 
10:42:59          	            	expected: {"counts":[1,0,1],"values":[0.045,0.29000000000000004,0.74]}
10:42:59          	            	actual  : {"counts":[0,0,1],"values":[0.045,0.29000000000000004,0.74]}
10:42:59          	            	
10:42:59          	            	Diff:
10:42:59          	            	--- Expected
10:42:59          	            	+++ Actual
10:42:59          	            	@@ -2,3 +2,3 @@
10:42:59          	            	  (string) (len=6) "counts": ([]uint64) (len=3) {
10:42:59          	            	-  (uint64) 1,
10:42:59          	            	+  (uint64) 0,
10:42:59          	            	   (uint64) 0,
10:42:59          	Test:       	TestPromHistogramToES/new_bucket_between_two_other_buckets_on_the_go
10:42:59      histogram_test.go:402: #2: {SampleCount:0x14000438c30 SampleSum:0x14000438c38 Bucket:[cumulative_count:2 upper_bound:0.09  cumulative_count:4 upper_bound:0.49  cumulative_count:16 upper_bound:0.99 ] XXX_NoUnkeyedLiteral:{} XXX_unrecognized:[] XXX_sizecache:0}
10:42:59      histogram_test.go:402: #3: {SampleCount:0x14000438c70 SampleSum:0x14000438c78 Bucket:[cumulative_count:3 upper_bound:0.09  cumulative_count:5 upper_bound:0.49  cumulative_count:18 upper_bound:0.99 ] XXX_NoUnkeyedLiteral:{} XXX_unrecognized:[] XXX_sizecache:0}
10:42:59      --- FAIL: TestPromHistogramToES/new_bucket_between_two_other_buckets_on_the_go (0.00s)
10:42:59  
10:42:59  === FAIL: x-pack/metricbeat/module/prometheus/collector TestPromHistogramToES (0.00s)
10:42:59  
10:42:59  DONE 408 tests, 7 skipped, 19 failures in 152.867s
10:42:59  Error: failed to execute go: exit status 1
```

See logs -> https://beats-ci.elastic.co/job/Beats/job/beats/job/PR-32173/5/execution/node/5333/log/

**Comment by v1v** [comment]
To unblock the task to enable the Orka M1 validations in the CI on a weekly basis -> https://github.com/elastic/beats/pull/32173

I just disabled the Orka M1 stage for `x-pack/metricbeat` with https://github.com/elastic/beats/pull/32173/commits/c47ccc51eb3fd3811336ecc2ce0c90a0ced8e545

Hence if https://github.com/elastic/beats/pull/32173 gets merged, you can enable back the support for MacOS m1 for `auditbeat` by uncommenting out the code in `x-pack/metricbeat/Jenkinsfile.yml` and attach the github label [macos-m1](https://github.com/elastic/beats/labels/macos-m1) , if it's not started the stage, please run the build again with `/test`

**Comment by oakrizan** [comment]
Closing, since it was related to Jenkins. New issue https://github.com/elastic/beats/issues/40496
